File: client/src/helper/calculateTotal.js
// Calculates total, determined by subtotal and shipping cost, owed by the customer
export default function calculateTotal(subtotal, shippingCost){
    let total = subtotal + shippingCost;

    // Rounds total to two decimal places
    let roundedTotal = Math.round(total * 100) / 100;

    return roundedTotal;
}

File: server/project/utils/calculate_total.py
# Calculates the total amount owed in pennies
# Args:
#   subtotal: Amount owed for the cart items
#   total: Sum of subtotal and shipping cost
# Return:
#   amount_in_pennies: Total amount owed in pennies
def calculate_total(subtotal, shipping_cost):
    # Initialize total amount owed
    total_amount = subtotal + shipping_cost

    # Rounds amount to two decimal places
    total_amount = round(total_amount, 2)

    return total_amount